Resources/Prototypes/_Scp/Entities/Mobs/Player/Scp/scp106_phantom.yml

```
@@ -8,6 +8,16 @@
     layers:
     - map: [ "base" ]
       state: phantom
+  - type: Damageable
+    damageContainer: Biological
+  - type: MobState
+    allowedStates:
+    - Alive
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 5

(indentation)